src/Pages/ReferenceTokens.razor

```diff
@@ -6,6 +6,7 @@
 @inject PersistedGrantDbContext persistedGrantDbContext
 @inject ConfigurationDbContext configurationDbContext
 @inject IReferenceTokenValidator referenceTokenValidator
+@inject IAuditLog databaseAuditLog
```

Comment:
Let's rename the local variable to `auditLog` as well so that this component no longer cares about where the data is actually stored.

Reply:
changed it in 5409f67